### PR TITLE
feat(docs): remove Font Awesome icons from examples

### PR DESCRIPTION
**Pull request by lucivpav**
**Overview**
There are examples in docs that use Font Awesome icons. If such an example snippet is run on e.g. Codesandbox, the icons are not rendered at all, due to missing Font Awesome. The Font Awesome should not be used in docs at all. Relevant issue #1762

**Changes**
1. <del>Font Awesome theme is no longer merged with `Teams` theme in `app.tsx` of docs</del>
2. All icon usages in code examples are from Teams theme
3. <del>GitHub icon has been added to `Teams` theme</del>
4. arrow-left, arrow-right were added to `Teams` theme (they are taken from `ProcessedIcons` folder)

<del>**TODO**
We need the following icons, that are currently missing from the `Teams` theme (they are also not in processedIcons):</del>
* <del>Copy, Copy outline</del>
* <del>Spinner</del>
* <del>Align right</del>
* <del>Minus</del>
* <del>Clock</del>
* <del>Thumbs down (aka Dislike)</del>
* <del>Folder</del>
* <del>Folder open</del>
* <del>User</del>

### CODE REVIEW

**Comment by levithomason** [comment]
Quick clarification, the Teams theme shouldn't use FontAwesome icons.  However, other themes may use FontAwesome icons.  It is also OK for the doc site itself to use FontAwesome icons.

I think only the actual examples files in `docs/src/examples` should avoid FontAwesome icons, and only for now while we work out icon name inconsistency issues.  However, this is also a temporary solution since switching the theme to anything other than Teams will also break once all the icons are switched to Teams.

Suggestions and reasoning:
- **Move forward with removing FontAwesome icon names from _examples_ only**
Just a temporary solution since the main theme being worked on is Teams right now.  This will have to updated again in the near future as soon as we start adding other themes.  Theme switching from Teams to something else should still work.

- **Leave FontAwesome support in Stardust**
Other themes likely will use it since it is the most popular and has a free license.  Most likely, will provide a base theme which also uses FontAwesome.

- **Leave the doc site on FontAwesome icons**
Stardust is not a Microsoft product but a general open source library.  We don't want to associate our docs with Microsoft's product glyphs.

**Review by levithomason — DISMISSED** [review]
See above

**Comment by lucivpav** [comment]
@levithomason I have updated this PR. Only the example code snippets are using Teams icons instead of Font Awesome icons. The usages of missing icons from Teams theme were replaced by other meaningful icons from Teams, so that we don't need to ask for new icons from the design team.

File: docs/src/examples/components/Popup/Variations/PopupExampleOffset.shorthand.tsx [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { Button, Grid, Popup } from '@stardust-ui/react'
 const renderButton = rotateArrowUp => (
   <Button
     icon={{
-      name: 'arrow circle up',
+      name: 'arrow-up',
+      circular: true,
```
Comment (miroslavstastny) [inline docs/src/examples/components/Popup/Variations/PopupExampleOffset.shorthand.tsx:8]: `circular` without `bordered` does not do anything visible. I would rather remove `circular`.
Also think about unifying Popup examples to use the same visual representation among examples.

File: docs/src/examples/components/Popup/Variations/PopupExamplePosition.shorthand.tsx [hunk 2]
```diff
@@ -53,7 +58,7 @@ const positionAndAlignValues = [
   'after-bottom',
 ]
 
-const icons: Record<Position, string> = {
+const iconNames: Record<Position, string> = {
   above: 'arrow circle up',
```
Comment (miroslavstastny) [inline docs/src/examples/components/Popup/Variations/PopupExamplePosition.shorthand.tsx]: These are still FA

File: docs/src/examples/components/Popup/Variations/PopupExampleOffset.shorthand.tsx [hunk 3]
```diff
@@ -4,7 +4,8 @@ import { Button, Grid, Popup } from '@stardust-ui/react'
...
       styles: { transform: `rotate(${rotateArrowUp})` },
```
Comment (miroslavstastny) [inline docs/src/examples/components/Popup/Variations/PopupExampleOffset.shorthand.tsx]: No need for styles, `Icon` has `rotate` prop

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts [hunk 4]
```diff
@@ -153,6 +155,8 @@ export default {
   add,
   'arrow-up': arrowUp,
   'arrow-down': arrowDown,
+  'arrow-left': arrowLeft,
```
Comment (miroslavstastny) [inline packages/react/src/themes/teams/components/Icon/svg/icons/index.ts:158]: Let only design add icons. Use `arrow-up` and rotate it as needed.

File: docs/src/views/Accessibility.tsx [hunk 5]
```diff
@@ -152,8 +152,8 @@ export default () => (
     <CodeSnippet
       value={`
         <>
-          <Button icon='envelope' />
-          <Button icon='envelope' aria-label='Send message' />
+          <Button icon='email' />
```
Comment (miroslavstastny) [inline docs/src/views/Accessibility.tsx:155]: As @levithomason mentioned - this is general doc, I would stick with FA here.

File: docs/src/examples/components/Popup/Variations/PopupExampleOffset.shorthand.tsx (hunk 1 above)
Comment (lucivpav) [reply docs/src/examples/components/Popup/Variations/PopupExampleOffset.shorthand.tsx:8]: In order to unify the Popup examples, I have decided to keep this icon `circular` and `bordered`.

File: packages/react/src/themes/teams/components/Icon/svg/icons/index.ts (hunk 4 above)
Comment (lucivpav) [reply packages/react/src/themes/teams/components/Icon/svg/icons/index.ts:158]: The two icons `arrow-left`, `arrow-right` are simply copied from `ProcessedIcons` folder. I did not design them.

File: docs/src/views/Accessibility.tsx (hunk 5 above)
Comment (lucivpav) [reply docs/src/views/Accessibility.tsx:155]: Are you sure? If user copies these code examples into the `codesandbox` template, the icons won't render.

Comment (miroslavstastny) [reply docs/src/views/Accessibility.tsx:155]: I am not sure, but let's go with what you have now.

**Review by miroslavstastny — APPROVED** [review]
Make sure you accept screen changes before merging.

**Pull request merged**